scanner/src/main/kotlin/ScannerCriteria.kt

```diff
@@ -108,6 +124,25 @@ data class ScannerCriteria(
                 configMatcher = exactConfigMatcher(details.configuration)
             )
         }
+
+        /**
+         * Return a [ScannerCriteria] object to be used when looking up existing scan results from a
+         * [ScanResultsStorage]. Per default, the properties of this object are initialized to match this scanner
+         * implementation. It is, however, possible to override these defaults from the configuration, in the
+         * [ScannerConfiguration.options] property: Use properties of the form `scannerName.property`, where
+         * `scannerName` is the name of the scanner the configuration applies to, and `property` is the name of a
+         * property of the [ScannerCriteria] class. For instance, to specify that a specific minimum version of ScanCode
+         * is allowed, set this property: `options.ScanCode.minVersion=3.0.2`.
+         */
+        fun fromConfig(details: ScannerDetails, config: ScannerConfiguration): ScannerCriteria {
```

Comment:
Now that this function has parameters, would it make sense to refer to them from the docs?

Reply:
Good point, I have improved the docs a bit.